afs-ws/afs-ws-server/src/main/java/com/powsybl/afs/ws/server/ExceptionHandler.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
```

Comment:
You should probably change the configuration of your IDE to use a variable for the year, instead of an hard coded value.